src/sentry/integrations/github/repository.py

```diff
@@ -43,6 +41,20 @@ def get_config(self, organization):
             }
         ]
 
+    def _get_repo(self, client, installation, repo):
```

Comment:
i'd maybe change this method name to something more explicit like `validate_repo` or `get_and_validate_repo`?